package.json

```diff
@@ -26,8 +26,8 @@
   "homepage": "https://github.com/okonet/metalsmith-webpack-dev-server#readme",
   "dependencies": {
     "chalk": "^1.1.1",
-    "metalsmith": "^1.7.0",
+    "metalsmith": "^2.2.0",
```

Comment:
There are changes that will break some configurations (especially around proxy rules due to an update in `node-http-proxy`).

I don't think they concern this package.